tst/standard/prop.tst

```diff
@@ -1028,6 +1028,222 @@ gap> g := Digraph([[2, 3], [3], [1]]);
+> , 87, 97, 114, 141, 151, 192, 211, 212, 222, 223 ], [ 2, 7, 224 ], [ 2, 3, 8, \
+> 12, 18, 148, 224, 225 ], [ 2, 4, 9, 23, 29, 39, 149, 190, 225, 226 ], [ 2, 5, \
+> 10, 44, 50, 60, 77, 150, 191, 211, 226, 227 ], [ 2, 6, 11, 82, 88, 98, 115, 14\
+> 2, 151, 192, 212, 223, 227, 228 ], [ 2, 3, 6, 174, 229 ], [ 3, 13, 19, 152, 22\
+> 5, 230 ], [ 3, 4, 14, 20, 24, 30, 35, 40, 153, 156, 160, 226, 230, 231 ], [ 3,\
+>  5, 15, 21, 45, 51, 56, 61, 69, 78, 157, 162, 171, 174, 196, 227, 231, 232 ], \
+> [ 3, 6, 16, 22, 83, 89, 94, 99, 107, 116, 129, 143, 158, 177, 197, 200, 213, 2\
+> 18, 228, 229, 232, 233 ], [ 2, 3, 4, 6, 34, 126, 129, 152, 175, 199, 213, 234 \
+> ], [ 2, 3, 4, 6, 129, 153, 175, 229, 235, 236 ], [ 2, 4, 5, 174, 198, 236 ], [\
+>  2, 3, 4, 6, 35, 199, 213, 229, 236, 237 ], [ 2, 4, 6, 229, 236, 238 ], [ 4, 2\
+> 5, 31, 41, 159, 161, 198, 231, 239 ], [ 4, 5, 26, 32, 42, 46, 52, 62, 68, 70, \
+> 79, 163, 164, 168, 172, 175, 199, 232, 236, 239, 240 ], [ 4, 6, 27, 33, 43, 84\
+> , 90, 100, 106, 108, 117, 127, 130, 144, 169, 178, 180, 201, 202, 207, 233, 23\
+> 4, 235, 237, 238, 240, 241 ], [ 2, 3, 4, 5, 6, 35, 66, 103, 124, 127, 152, 153\
+> , 159, 160, 161, 173, 201, 242 ], [ 2, 3, 4, 5, 6, 17, 35, 65, 105, 127, 130, \
+> 152, 153, 160, 163, 176, 234, 243, 244 ], [ 2, 3, 5, 67, 152, 175, 199, 244 ],\
+>  [ 2, 3, 4, 5, 6, 17, 35, 55, 68, 123, 152, 153, 160, 176, 201, 202, 234, 244,\
+>  245 ], [ 2, 3, 5, 69, 171, 175, 199, 236, 246 ], [ 2, 3, 5, 6, 67, 126, 152, \
+> 235, 237, 246, 247 ], [ 2, 3, 5, 6, 129, 171, 235, 238, 246, 248, 250 ], [ 2, \
+> 3, 4, 5, 6, 56, 70, 176, 200, 202, 237, 246, 249 ], [ 2, 5, 236, 250 ], [ 2, 3\
+> , 5, 6, 69, 213, 237, 238, 246, 250, 251 ], [ 2, 5, 6, 238, 250, 252 ], [ 5, 4\
+> 7, 53, 63, 80, 170, 173, 176, 240, 244, 246, 250, 253 ], [ 2, 3, 4, 5, 6, 107,\
+>  130, 162, 164, 176, 235, 246, 254 ], [ 5, 6, 48, 54, 64, 81, 85, 91, 101, 118\
+> , 125, 128, 131, 145, 179, 181, 182, 187, 241, 242, 243, 245, 247, 248, 249, 2\
+> 51, 252, 253, 254, 255 ], [ 2, 3, 4, 5, 6, 35, 56, 68, 122, 125, 152, 153, 159\
+> , 162, 163, 170, 179, 256 ], [ 2, 3, 4, 5, 6, 28, 35, 36, 38, 68, 69, 104, 121\
+> , 125, 128, 152, 153, 160, 161, 163, 171, 172, 179, 181, 242, 257 ], [ 2, 3, 4\
+> , 6, 35, 103, 124, 152, 153, 159, 160, 161, 198, 243, 245, 258 ], [ 2, 3, 4, 5\
+> , 6, 17, 28, 35, 36, 56, 69, 93, 106, 120, 128, 131, 152, 153, 160, 162, 171, \
+> 174, 178, 181, 182, 243, 245, 259 ], [ 2, 3, 4, 6, 17, 35, 105, 123, 127, 152,\
+>  153, 160, 201, 247, 249, 254, 260 ], [ 2, 3, 6, 126, 152, 248, 251, 261 ], [ \
+> 2, 3, 4, 5, 6, 94, 108, 131, 177, 180, 182, 249, 254, 262 ], [ 2, 3, 4, 6, 107\
+> , 130, 200, 202, 248, 249, 251, 254, 263 ], [ 2, 3, 6, 129, 213, 248, 251, 252\
+> , 264 ], [ 2, 6, 252, 265 ], [ 6, 86, 92, 102, 119, 146, 255, 256, 257, 258, 2\
+> 59, 260, 261, 262, 263, 264, 265, 266 ] ] );
+<digraph with 266 vertices, 3121 edges>
+gap> gr := DigraphReflexiveTransitiveClosure(gr);
```

Comment:
Also, on reflection, I'm not sold on the purpose of this test. It is mostly testing `IsPartialOrderDigraph` - but since by construction the digraph is reflexive and transitive, the only thing it is checking is `IsAntisymmetricDigraph`. And since this digraph is neither a meet nor a join semilattice, that code isn't really being stretched either – presumably the methods there deduce almost immediately that the thing isn't a semilattice.

If you're going to go to the effort of putting such a large and fiddling test into the test file, I'd suggest doing one which is actually going to exhibit interest behaviour. So, for instance, one which is maybe a meet semilattice, but not a join semilattice. This way you're testing the code much more thoroughly.